src/core/import/qImport.ts

```diff
@@ -20,19 +18,14 @@ declare const __mockImport: (path: string) => Promise<any>;
  * @param base -`QRL`s are relative, and therefore they need a base for resolution.
  *    - `Element` use `base.ownerDocument.baseURI`
  *    - `Document` use `base.baseURI`
- *    - `string` use `base` as is
- *    - `QConfig` use `base.baseURI`
  * @param url - A relative URL (as `string` or `QRL`) or fully qualified `URL`
  * @returns A cached value synchronously or promise of imported value.
  * @public
  */
-export function qImport<T>(
-  base: Element | Document | string | QConfig,
-  url: string | QRL<T> | URL
-): T | Promise<T> {
+export function qImport<T>(base: Element | Document, url: string | QRL<T> | URL): T | Promise<T> {
   if (!importCache) importCache = new Map<string, unknown | Promise<unknown>>();
```

Comment:
As it stands now, I think this will not behave correctly. On a server, one can serve different apps. These apps will have different configurations but may have the same protocol. The above will leak data between the applications. Let's remove this cache altogether.